README.md

```diff
@@ -91,14 +115,99 @@ For **Azure CLI**, create a `public.json` (and optionally `protected.json`) and
 
 
 
+_For PowerShell with variables_
+ All PowerShell examples assume you are successfully connected to an Azure subscription. 
+   For more information see: http://itproguru.com/expert/2016/04/powershell-working-with-azure-resource-manager-rm-step-by-step-changing-rm-subscriptions/ 
```

Comment:
I'm not sure we're cleared to endorse 3rd party tutorials cc: @neilpeterson what do you think?